Mon Sep 25 21:25:55 2023
macOS-13.2-arm64-arm-64bit
Ren'Py 8.1.2.23090503
 

Bootstrap to the start of init.init. took 0.02s
Early init. took 0.00s
Loader init. took 0.01s
Loading error handling. took 0.02s
Loading script. took 0.07s
Loading save slot metadata. took 0.05s
Loading persistent. took 0.00s
Running init code. took 0.03s
Loading analysis data. took 0.01s
Analyze and compile ATL. took 0.01s
Reloading save slot metadata. took 0.02s
Index archives took 0.00s
Dump and make backups. took 0.03s
Cleaning cache. took 0.00s
Making clean stores. took 0.00s
Initial gc. took 0.02s
DPI scale factor: 1.000000
Creating interface object. took 0.00s
Cleaning stores took 0.00s
Init translation took 0.03s
Build styles took 0.00s
Load screen analysis took 0.02s
Analyze screens took 0.00s
Save screen analysis took 0.00s
Prepare screens took 0.04s
Save pyanalysis. took 0.00s
Save bytecode. took 0.00s
Running _start took 0.00s
Performance test:
Interface start. took 0.26s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1440, 900)
swap interval: 1 frames
Windowed mode.
Vendor: "b'Apple'"
Renderer: b'Apple M1'
Version: b'2.1 Metal - 83'
Display Info: None
Screen sizes: virtual=(1920, 1080) physical=(769, 433) drawable=(1538, 866)
Maximum texture size: 4096x4096

Full traceback:
  File "renpy/common/_layout/screen_main_menu.rpym", line 28, in script
    python hide:
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/_layout/screen_main_menu.rpym", line 28, in <module>
    python hide:
  File "renpy/common/_layout/screen_main_menu.rpym", line 35, in _execute_python_hide
    ui.interact()
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/display/core.py", line 3587, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/display/core.py", line 4384, in interact_core
    renpy.audio.audio.periodic()
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/audio/audio.py", line 1224, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 444, in raise_
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/audio/audio.py", line 1202, in periodic_thread_main
    periodic_pass()
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/audio/audio.py", line 1144, in periodic_pass
    c.periodic()
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/audio/audio.py", line 534, in periodic
    topf = load(filename)
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/audio/audio.py", line 72, in load
    rv = renpy.loader.load(fn, directory="audio")
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/loader.py", line 833, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'Raudio/Music/A_Trick_of_Mind_Menu.ogg'.

While running game code:
OSError: Couldn't find file 'Raudio/Music/A_Trick_of_Mind_Menu.ogg'.
Resetting cache.
Bootstrap to the start of init.init. took 16.21s
Early init. took 0.00s
Loader init. took 0.01s
Loading error handling. took 0.02s
Loading script. took 0.11s
Loading save slot metadata. took 0.04s
Loading persistent. took 0.00s
Running init code. took 0.03s
Loading analysis data. took 0.01s
Analyze and compile ATL. took 0.01s
Reloading save slot metadata. took 0.03s
Index archives took 0.00s
Backing up script files to '/Users/namseoa/Library/RenPy/backups/A_Trick_Of_Mind':
Dump and make backups. took 0.03s
Cleaning cache. took 0.00s
Making clean stores. took 0.00s
Initial gc. took 0.04s
DPI scale factor: 1.000000
Creating interface object. took 0.00s
Cleaning stores took 0.00s
Init translation took 0.03s
Build styles took 0.00s
Load screen analysis took 0.00s
Analyze screens took 0.01s
Save screen analysis took 0.01s
Prepare screens took 0.04s
Save pyanalysis. took 0.00s
Save bytecode. took 0.01s
Running _start took 0.00s
Performance test:
Interface start. took 0.09s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1440, 900)
swap interval: 1 frames
Windowed mode.
Vendor: "b'Apple'"
Renderer: b'Apple M1'
Version: b'2.1 Metal - 83'
Display Info: None
Screen sizes: virtual=(1920, 1080) physical=(769, 433) drawable=(1538, 866)
Maximum texture size: 4096x4096

Full traceback:
  File "renpy/common/00start.rpy", line 212, in script call
    call _load_reload_game from _call__load_reload_game_1
  File "renpy/common/00keymap.rpy", line 529, in script
    python hide:
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00keymap.rpy", line 529, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 540, in _execute_python_hide
    renpy.load(renpy.session["_reload_slot"])
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/loadsave.py", line 816, in load
    log.unfreeze(roots, label="_after_load")
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/rollback.py", line 1108, in unfreeze
    self.rollback(0, force=True, label=label, greedy=greedy, on_load=True)
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/rollback.py", line 908, in rollback
    self.load_failed()
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/rollback.py", line 826, in load_failed
    raise Exception("Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?")
Exception: Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?

While running game code:
  File "renpy/common/00start.rpy", line 212, in script call
    call _load_reload_game from _call__load_reload_game_1
  File "renpy/common/00keymap.rpy", line 529, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 529, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 540, in _execute_python_hide
    renpy.load(renpy.session["_reload_slot"])
Exception: Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?
